### PR TITLE
UML-3021: Settings page changes for GOV.UK One Login

### PR DESCRIPTION
**Pull request by Lbagg1**
# Purpose

Update the settings page as shown in the design attached.

AC

Behind one login feature flag. (not a deletion, but a hiding)

========================================================================

Settings page header

Title:

Settings

========================================================================

Settings page / Your sign-in details

Title:

Your sign-in details

Content:

You use GOV.UK One Login to sign in to the Use a lasting power of attorney service.

You can change these details in your GOV.UK One Login:

email address

password

how you get security codes to sign in

Change your sign-in details in your GOV.UK One Login

========================================================================

Settings page / Delete your lasting power of attorney account

Title:

Delete your Use a lasting power of attorney account

Content:

If you no longer need your Use a lasting power of attorney account, you can delete it.

Find out more about deleting your account
========================================================================

Acceptance criteria:

Behind flag

New content

Change your sign-in details in your [Gov.uk](http://gov.uk/) One Login  links to https://home.account.gov.uk/security

[UML-3021](https://opgtransform.atlassian.net/jira/software/c/projects/LPA/boards/149?assignee=712020%3A769cea9b-19b5-46c8-bb8d-ca2203316228&selectedIssue=UML-3021)
## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [x] I have performed a self-review of my own code
~~* [ ] I have added relevant logging with appropriate levels to my code~~
~~* [ ] New event_codes have been documented on the wiki page]~~~~(https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant ~~
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
~~* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found~~
* [x] I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made
* [ ] The product team have tested these changes

[UML-3021]: https://opgtransform.atlassian.net/browse/UML-3021?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: service-front/app/features/actor-delete-account.feature
Comment (cooperaj) [inline service-front/app/features/actor-delete-account.feature:1]: I think account deletions are still a thing even with one login?

File: service-front/app/features/actor-your-details.feature
Comment (cooperaj) [inline service-front/app/features/actor-your-details.feature:1]: And the details link/page should also still work for one login?

File: service-front/app/src/Actor/templates/actor/settings.html.twig [hunk 1]
```diff
@@ -11,45 +11,87 @@
...
+                            </div>
+                            <div class="govuk-summary-card__content">
+                                <p class="govuk-body">{% trans %}If you no longer use your Use a lasting power of attorney account, you can delete it.{% endtrans %}</p>
+                                <a href="{{ path('confirm-delete-account') }}" class="govuk-link">
+                                    {% trans %}Find out more about deleting your account{% endtrans %}
+                                </a>
+                            </div>
+                        </div>
+                    {% else %}
+                        <div class="govuk-summary-card">
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/settings.html.twig:45]: Does this mean the design of the currently live page is changed? I thought this PR should just introduce the new page if the flag was on?

File: service-front/app/features/actor-delete-account.feature
Comment (Lbagg1) [reply service-front/app/features/actor-delete-account.feature:1]: I was just checking something - I'm reverting this

**Review by cooperaj — COMMENTED** [review]
Looks great, minor UAT feedback would be that the links in the boxes are a smaller text size than the body text and that's not the case on the Figma mockup.

I think this should be as easy to fix as adding `govuk-body` to the links class?

**Review by cooperaj — DISMISSED** [review]
LGTM

**Review by nickdavis2001 — DISMISSED** [review]

**Review by nickdavis2001 — APPROVED** [review]

**Pull request merged**